Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/warn/list/items/TraceLocationVH.kt

```diff
@@ -29,25 +31,25 @@ class TraceLocationVH(parent: ViewGroup) :
 
         if (item.traceLocation.startDate != null && item.traceLocation.endDate != null) {
 
-            val startTime = item.traceLocation.startDate.toDateTime()
-            val endTime = item.traceLocation.endDate.toDateTime()
+            val startTime = item.traceLocation.startDate.toLocaleDateTimeUtc()
+            val endTime = item.traceLocation.endDate.toLocaleDateTimeUtc()
```

Comment:
This is for the UI I assume. Then how come we convert to UTC instead of user time?